spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/ContinueSpan.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Makes sense, just wondering if the spring-data folks would add tags/spans on things in the main data modules if it meant they didn't have to bring in all of sleuth